src/libraries/System.Globalization/tests/CompareInfo/CompareInfoTests.Compare.cs

```diff
@@ -41,78 +41,79 @@ public static IEnumerable<object[]> Compare_Kana_TestData()
-            yield return new object[] { s_invariantCompare, "\u30FC", "\u30FC", ignoredOptions, 0 };
-            yield return new object[] { s_invariantCompare, "\u30FC", "\u2015", ignoredOptions, 1 };
-            yield return new object[] { s_invariantCompare, "\u30FC", "\u2010", ignoredOptions, 1 };
-
-            yield return new object[] { s_invariantCompare, "/", "\uFF0F", ignoredOptions, 0 }; // known exception for hg: should be -1
-            yield return new object[] { s_invariantCompare, "'", "\uFF07", ignoredOptions, 0 }; // as above
-            yield return new object[] { s_invariantCompare, "\"", "\uFF02", ignoredOptions, 0 }; // as above
-
-            yield return new object[] { s_invariantCompare, "\u3042", "\u30A1", CompareOptions.None, s_expectedHiraganaToKatakanaCompare };
+            if (!PlatformDetection.IsHybridGlobalizationOnBrowser)
+            {
+                yield return new object[] { s_invariantCompare, "a", "\uFF41", CompareOptions.IgnoreWidth, 0 };
+                yield return new object[] { s_invariantCompare, "ABCDE", "\uFF21\uFF22\uFF23\uFF24\uFF25", CompareOptions.IgnoreWidth, 0 };
+                yield return new object[] { s_invariantCompare, "ABCDE", "\uFF21\uFF22\uFF23D\uFF25", CompareOptions.IgnoreWidth, 0 };
+                yield return new object[] { s_invariantCompare, "ABCDE", "a\uFF22\uFF23D\uFF25", CompareOptions.IgnoreWidth | CompareOptions.IgnoreCase, 0 };
+                yield return new object[] { s_invariantCompare, "ABCDE", "\uFF41\uFF42\uFF23D\uFF25", CompareOptions.IgnoreWidth | CompareOptions.IgnoreCase, 0 };
+                yield return new object[] { s_invariantCompare, "ABCDE", "\uFF43D", CompareOptions.IgnoreWidth, -1 };
+                yield return new object[] { s_invariantCompare, "ABCDE", "c", CompareOptions.IgnoreWidth, -1 };
+                yield return new object[] { s_invariantCompare, "\u30BF", "\uFF80", CompareOptions.IgnoreWidth, 0 };
+                yield return new object[] { s_invariantCompare, "0", "\uFF10", CompareOptions.IgnoreWidth, 0 };
+                yield return new object[] { s_invariantCompare, "10", "1\uFF10", CompareOptions.IgnoreWidth, 0 };
+                yield return new object[] { s_invariantCompare, "\uFF1B", ";", CompareOptions.IgnoreWidth, 0 };
+                yield return new object[] { s_invariantCompare, "\uFF08", "(", CompareOptions.IgnoreWidth, 0 };
+                yield return new object[] { s_invariantCompare, "/", "\uFF0F", CompareOptions.IgnoreWidth, 0 };
+                yield return new object[] { s_invariantCompare, "'", "\uFF07", CompareOptions.IgnoreWidth, 0 };
+                yield return new object[] { s_invariantCompare, "\"", "\uFF02", CompareOptions.IgnoreWidth, 0 };
+            }
+            yield return new object[] { s_invariantCompare, "\u3042", "\u30A1", CompareOptions.None, PlatformDetection.IsHybridGlobalizationOnOSX ? 1 : s_expectedHiraganaToKatakanaCompare };
```

Comment:
Is the ordering native apple platforms with `NSLiteralSearch`:
`lowercase hiragana < lowercase katakana < uppercase hiragana < uppercase katakana` ?
I believe this should be noted in `hybrid-globalization.md` as the comments there suggest `hiragana and katakana characters are ordered differently compared to ICU` but it seems to depend on the case as well?

I am curious if `\u3042` `\u30A1` `CompareOptions.IgnoreCase` leads to `-1`.

If this is the case, I think in addition to noting the behavior in the docs, it would be nice to organize the hiragana/katakana tests to better reflect the different cases (i.e. Same case hiragana vs katakana, lower case hiragana vs upper case katakana, upper case hiragana vs lower case katakana or something to highlight expected behaviors) Test changes can be done in a followup, but I believe documentation change should be added in this PR.

Reply:
It does not depend on case. Only different behaviour for `IgnoreCase` was comparing `\u3060` だ  with `\u30C0` ダ and as `hiragana uppercase` is after `katakana uppercase` it gives `1` instead of `-1` (for `ICU`). 

Ordering in native apple platforms is like `hiragana lowercase < katakana lowercase < katakana uppercase < hiragana uppercase`. Updated in the doc and added 3 test cases showing this ordering.